src/utils/config.ts

```diff
@@ -1,13 +1,21 @@
-export const enum DecklistPosition {
+export const enum OverlayPosition {
 	TOP_LEFT = "topleft",
+	TOP_CENTER = "topcenter",
 	TOP_RIGHT = "topright",
 }
 
+export const enum WhenToShowBobsBuddy {
+	All = "all",
+	OnlyInShopping = "onlyinshopping",
+	OnlyInCombat = "onlyincombat",
+}
+
 export const enum Feature {
 	DECKLIST = 1,
 	TOOLTIPS = 2,
-	// C = 4,
-	// D = 8,
+	BOBSBUDDY = 4,
+	// C = 8,
+	// D = 16,
```

Comment:
We should write these as
```
DECKLIST = 1 << 0
TOOLTIPS = 1 << 1,
BOBSBUDDY = 1 << 2
```
to indicate these are flags and then we don't need the C/D comments
```